models/topic.go

```diff
@@ -20,6 +21,8 @@ func init() {
 	)
 }
 
+var topicPattern = regexp.MustCompile(`^[a-z0-9+#_.-]+$`)
```

Comment:
I think we should limit same as github does with regexp: `^[a-z0-9][a-z0-9-]*$`

Reply:
Ok, Changed it